core/arxiv/submission/services/classic/tests/test_get_submission.py

```diff
@@ -157,3 +157,93 @@ def test_get_submission_with_hold_and_reclass(self):
         self.assertTrue(submission_loaded.is_on_hold,
                         "Hold status should reflect hold action performed"
                         " outside the purview of the event model.")
+
+    def test_get_submission_list(self):
+        """Test that the set of submissions for a user can be retrieved."""
+        user = User(42, 'adent@example.org',
+                    endorsements=['astro-ph.GA', 'astro-ph.EP'])
+        events1 = [
+            # first submission
+            CreateSubmission(creator=user),
+            SetTitle(creator=user, title='Foo title'),
+            SetAbstract(creator=user, abstract='Indeed' * 20),
+            SetAuthors(creator=user, authors=[
+                Author(order=0, forename='Arthur', surname='Dent',
+                       email='adent@example.org'),
+                Author(order=1, forename='Ford', surname='Prefect',
+                       email='fprefect@example.org'),
+            ]),
+            SetLicense(creator=user, license_uri='http://creativecommons.org/publicdomain/zero/1.0/',
+                       license_name='Foo zero 1.0'),
+            SetPrimaryClassification(creator=user, category='astro-ph.GA'),
+            ConfirmPolicy(creator=user),
+            SetUploadPackage(creator=user, identifier='1'),
+            ConfirmContactInformation(creator=user),
+            FinalizeSubmission(creator=user)
+        ]
+        events2 = [
+            # second submission
+            CreateSubmission(creator=user),
+            SetTitle(creator=user, title='Bar title'),
+            SetAbstract(creator=user, abstract='Indubitably' * 20),
+            SetAuthors(creator=user, authors=[
+                Author(order=0, forename='Jane', surname='Doe',
+                       email='jadoe@example.com'),
+                Author(order=1, forename='John', surname='Doe',
+                       email='jodoe@example.com'),
+            ]),
+            SetLicense(creator=user, license_uri='http://creativecommons.org/publicdomain/zero/1.0/',
+                       license_name='Foo zero 1.0'),
+            SetPrimaryClassification(creator=user, category='astro-ph.GA'),
+            ConfirmPolicy(creator=user),
+            SetUploadPackage(creator=user, identifier='1'),
+            ConfirmContactInformation(creator=user),
+            FinalizeSubmission(creator=user)
+        ]
+
+        # TODO: create "classic", non-event-derived submission
```

Comment:
The test I started writing here might be more useful if there's a classic submission in the mix.